test/unit/uk/gov/hmrc/apinotificationpull/controllers/NotificationsControllerSpec.scala

```diff
@@ -86,6 +98,17 @@ class NotificationsControllerSpec extends UnitSpec with WithFakeApplication with
 
       result shouldBe presentedNotification
     }
+
```

Comment:
As this is a controller test I think this should mention 500/INTERNAL_SERVER_ERROR in test name rather than `fail if ApiNotificationQueueService failed` . Same applies for test for `getAll`